src/Routes/index.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import { Switch, Route } from 'react-router-dom';
 import Home from '../components/Home';
+import Counter from '../containers/Counter/Counter';
 
 function index() {
   return (
     <Switch>
+      <Route exact path="/counter" component={Counter} />
```

Comment:
i think we do need one exact path

Reply:
`exact` disables the partial matching for a route. 
Here will return the route if only the path is exact match to this URL